### ISSUE TITLE
Mods didn't load

### ISSUE THREAD

**Issue by EnderDragon25** [ZaneYork/SMAPI-Android-Installer#701]
Hello. Maybe I'm dumb but i got the same issue mentioned here https://github.com/ZaneYork/SMAPI-Android-Installer/issues/590 and downloaded both patched stardew valley and stardew valley.zip. After unpacking and installing apk everything worked fine. But not the mods. In smapi i set mods folder to stardew valley/mods and dropped everything there, however when launching smapi it loads only 2 default mods. Still giving errors when mods can't load or need dependencies.

**Comment by bai0012** [comment]
It is possible that the Mod folder is not located in "/storage/emulated/0/StardewValley/Mods" as we originally thought, but rather at "/storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods" (using my phone as an example). 

You can open Stardew Valley with SMAPI, after entering the main interface, use the virtual keyboard to access the console interface. The second line of the console will tell us where the mod folder is, for example: 

[SMAPI] Mods go here: /storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods

Move your mods there and see if they are loaded correctly.

**Comment by EnderDragon25** [comment]
> It is possible that the Mod folder is not located in "/storage/emulated/0/StardewValley/Mods" as we originally thought, but rather at "/storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods" (using my phone as an example).
> 
> You can open Stardew Valley with SMAPI, after entering the main interface, use the virtual keyboard to access the console interface. The second line of the console will tell us where the mod folder is, for example:
> 
> [SMAPI] Mods go here: /storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods
> 
> Move your mods there and see if they are loaded correctly.

yeah that's what i thought. Pretty lame you need to use PC for the access to the Android/data but i'm gonna try

**Comment by Jeckabnormalism** [comment]
> > It is possible that the Mod folder is not located in "/storage/emulated/0/StardewValley/Mods" as we originally thought, but rather at "/storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods" (using my phone as an example).
> > You can open Stardew Valley with SMAPI, after entering the main interface, use the virtual keyboard to access the console interface. The second line of the console will tell us where the mod folder is, for example:
> > [SMAPI] Mods go here: /storage/emulated/0/Android/data/com.zane.stardewvalley/files/Mods
> > Move your mods there and see if they are loaded correctly.
> 
> yeah that's what i thought. Pretty lame you need to use PC for the access to the Android/data but i'm gonna try

Use ZArchiver! It helps!
